src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()"
+    >
```

Comment:
3rd item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) make sure to fix it everywhere
```suggestion
    <form 
    	action="https://mate-academy-form-lesson.herokuapp.com/create-application"
    	method="post"
    	onsubmit="onSubmit()"
    >
```